addons/vehicleDamage/XEH_postInit.sqf

```diff
@@ -0,0 +1,63 @@
+#include "script_component.hpp"
+#include "\z\ace\addons\medical_engine\script_macros_medical.hpp"
```

Comment:
because of this include we'll need to add to required addons,
I think we just need this for `IS_UNCON` macro?
might be simpler to switch to
 `ace_common_fnc_isAwake`: `alive _unit && {!(_unit getVariable ["ACE_isUnconscious", false])}`